free-1.md

```diff
@@ -1,50 +1,61 @@
-# Free project 1
+# Free project 1: Music and Visual Art generated from keystrokes or text
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This project could be used by anyone looking to create animations and musical notes from within their web browser. 
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+There isn't a particular need that the project would fulfill. Perhaps an innate "need to create art in a spontaneous and fun way and be able to share it with others". While a program like ContextFree is very methodical and has quite a steep learning curve, this project would allow users to be creative in a much easier and more engaging way while retaining the element of randomness that makes ContextFree interesting. 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+N/A
```

Comment:
You should still address the domain being targeted and why a DSL is a good way to reach that domain.